### PR TITLE
[Blueprints] Remove resource group name and location length validation in a blueprint

### PR DESCRIPTION
**Pull request by filizt**

### CODE REVIEW

**Comment by filizt** [comment]
Autorest removed the [Newtonsoft.Json.JsonObject("Artifact")] attribute.  @shenglol Looking at the history it seems like this attribute was not there, got added during the MG level assignment SDK updates and now when I run autorest to generate the SDK it's removed again. Do you have any pointer why this could have happened?

File: sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
 [assembly: AssemblyVersion("0.20.0.0")]
-[assembly: AssemblyFileVersion("0.20.6.0")]
+[assembly: AssemblyFileVersion("0.20.8.0")]
```
Comment (isra-fel) [inline sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs:15]: Not sure why 0.20.7 was skipped 🤔 ; besides please also update the version in .csproj

**Review by isra-fel — CHANGES_REQUESTED** [review]

**Comment by shenglol** [comment]
> Autorest removed the [Newtonsoft.Json.JsonObject("Artifact")] attribute. @shenglol Looking at the history it seems like this attribute was not there, got added during the MG level assignment SDK updates and now when I run autorest to generate the SDK it's removed again. Do you have any pointers for why this could have happened?

A possible reason I can think of is that AutoRest loaded different versions of the `@microsoft.azure/autorest.csharp` and `@microsoft.azure/autorest.modeler` extensions at the time we generated the SDK, though we seemed to use the same AutoRest version. But I think we should be fine here, since adding `JsonObject` and removing it are identical - all properties of `Artifact` already have `JsonProperty` attributes defined.

File: sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (filizt) [reply sdk/blueprint/Microsoft.Azure.Management.Blueprint/src/Properties/AssemblyInfo.cs]: Addressed.

**Review by isra-fel — CHANGES_REQUESTED** [review]
LGTM

**Review by isra-fel — APPROVED** [review]

**Pull request merged**